### DISCUSSION TITLE
bullet3d fix for clang

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#9252] [General]
[install-openvr-only-msvc.zip](https://github.com/msys2/MINGW-packages/files/6910388/install-openvr-only-msvc.zip)

corrected patch, now builds on clang64 also.
remember to disable numpy for clang builds.

**Comment by jeremyd2019** [comment]
I got numpy building on clang, fortran and BLAS are optional for numpy.

**Reply by revelator** [reply]
oh ? nwm then :)

**Comment by revelator** [comment]
incidently have you had any luck getting postgres to build with clang ?, i had nothing but trouble fixing one problem just leads to several others.

**Reply by jeremyd2019** [reply]
#9167

**Comment by revelator** [comment]
p.s patch is still good though, as for some reason clangs linker tries to link to the prebuilt openvr library which is built with msvc. so i updated the existing patch so that this particular example newer gets build, the original patch only disabled installing it.

**Reply by Biswa96** [reply]
Your reply is packed with bunch of metadata garbage.

**Comment by revelator** [comment]
oups i was not at home, reply sent via my phone but i see now.

only mentioned that the patch is still ok to use as clang tries to link in the prebuilt msvc openvr libraries and the original patch only disabled installing the example for it, so i disabled building the faulty example.

**Reply by revelator** [reply]
forgot need to add

```
  # clang complains about narrowing issues and aborts.
  CFLAGS+=" -Wno-narrowing"
  CXXFLAGS+=" -Wno-narrowing"
```

or it fails to build
